packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -199,7 +199,7 @@ const DomainStep: React.FunctionComponent< DomainStepProps > = ( {
 						<p className="focused-launch-summary__side-commentary-title">
 							{ createInterpolateElement(
 								__(
-									'<strong>Unique domains</strong> help build brand recongnition and trust',
+									'<strong>Unique domains</strong> help build brand recognition and trust',
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **1** times:
`translate( '<strong>Unique domains</strong> help build brand trust' )` *ES Score: 8*